File: Arrays/Sort-Zeros-Ones-Twos/Sort-Zeros-Ones-Twos.js
function SortZeroOneTwo(arr){
    for(let i=0; i<arr.length; i++){
        for(let j=i+1; j<arr.length; j++){
            if(arr[i] > arr[j]){
                [arr[i], arr[j]] = [arr[j], arr[i]]
            }
        }
    }
    return arr
}

let arr = [1, 0, 2, 1, 2, 1, 0, 2, 0, 1, 2, 0, 0, 1, 1, 2, 2]
const result = SortZeroOneTwo(arr)
console.log(result);

File: Arrays/Sort-Zeros-Ones-Twos/sort_zeros_ones_twos.py
def sort_zeros_ones_twos(arr):
    for i in range(0, len(arr)):
        for j in range(i+1, len(arr)):
            if(arr[i] > arr[j]):
                arr[i], arr[j] = arr[j], arr[i]
    return arr


arr = [1, 0, 2, 1, 2, 1, 0, 2, 0, 1, 2, 0, 0, 1, 1, 2, 2]
print(sort_zeros_ones_twos(arr))